shellbase-core/src/main/scala/com/sumologic/shellbase/ShellCommand.scala

```diff
@@ -143,7 +156,7 @@ abstract class ShellCommand(val name: String,
     }
   }
 
-  def postCommandToSlack(cmdPath: List[String], args: List[String]): Option[String] = {
+  def postCommandToSlack(cmdPath: List[String], args: List[String], comments: Option[String]): Option[String] = {
```

Comment:
Also a breaking change.  At the very least, I'd consider a default value here

Reply:
The default value wouldn't help with overriding methods anyway.
What actually is an option - is to add an overloaded variant with and without the `comments` argument, but that only adds complexity with little gain, in my mind.